### PR TITLE
Update readme instructions regarding installation

### PR DESCRIPTION
**Pull request by cdrage**
Updates the readme for how to install Kompose correctly.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,31 +14,42 @@ and [examples](https://github.com/redhat-developer/opencompose/blob/master/examp
...
+
+#### Binary installation
+
 You can retrieve binaries for Linux, macOS and Windows on our [GitHub release page](https://github.com/redhat-developer/opencompose/releases).
 
-##### Linux
-If you prefer to do it straight from CLI here's a one-liner for you:
-```bash
+``sh
+# Linux
```
Comment (pradeepto) [inline README.md]: @cdrage These don't render correctly. The # Linux breaks it, at least in the preview mode. Same applies below as well.

**Review by pradeepto — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (surajnarwade) [reply README.md]: one  backtick character is missing on line 23, that's why its breaking

Comment (surajnarwade) [inline README.md]: one backtick character is missing on line 23, that's why its breaking

Comment (cdrage) [reply README.md]: Woops. I'll quickly fix that.

**Comment by cdrage** [comment]
@pradeepto @surajnarwade Done. Ready for another quick review.

**Comment by surajssd** [comment]
cool this LGTM!

File: README.md [hunk 2]
```diff
@@ -14,31 +14,42 @@ and [examples](https://github.com/redhat-developer/opencompose/blob/master/examp
...
-```bash
+```sh
+# Linux 
 curl -L https://github.com/redhat-developer/opencompose/releases/download/v0.1.0/opencompose-linux-amd64 -o opencompose
+
+# macOS
+curl -L https://github.com/redhat-developer/opencompose/releases/download/v0.1.0/opencompose-darwin-amd64 -o opencompose
+
+# Windows
+curl -L https://github.com/redhat-developer/opencompose/releases/download/v0.1.0/opencompose-windows-amd64.exe -o opencompose.exe
```
Comment (kadel) [inline README.md:31]: is `curl` on windows?

Comment (cdrage) [reply README.md:31]: According to `kubectl`, yup. Binary installation, works well.

**Pull request merged**